### ISSUE TITLE
doubts about cross entropy with softmax

### ISSUE THREAD

**Issue by Tveek** [unsky/focal-loss#21]
why CE is not this:
![ce](https://user-images.githubusercontent.com/14250619/41197551-37ef09d4-6c95-11e8-8331-f464722200ca.jpg)
by my formula,derivatives is different from yours.
